apko-snapshot/action.yaml

```diff
@@ -184,6 +184,21 @@ runs:
         username: ${{ inputs.username }}
         password: ${{ inputs.token }}
 
+    # Upload the SBOMs as attestations
+    # TODO: instrument apko to produce these on disk so we are not downloading them from remote
```

Comment:
This comment sort of buries the lede.

This approach fundamentally trusts the registry to return to us the SBOM we just uploaded, which is a weak link in the integrity of our metadata chain.  We are signing these, so IMO we should know that we're signing what we think we're signing.

Reply:
what is the suggestion here?